pkg/target/manager_test.go

```diff
@@ -461,36 +446,36 @@ var _ = Describe("Manager", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(manager).NotTo(BeNil())
 
-		res, dropErr := manager.DropTargetProject()
-		Expect(dropErr).To(HaveOccurred())
+		res, unsetErr := manager.UnsetTargetProject()
+		Expect(unsetErr).To(HaveOccurred())
 		Expect(res).To(BeEmpty())
 		assertTargetProvider(targetProvider, t)
 	})
 
-	It("should unset deeper target levels when dropping project", func() {
-		t := target.NewTarget(gardenName, prod1Project.Name, seed.Name, prod1AmbiguousShoot.Name)
+	It("should unset deeper target levels when unsetting project", func() {
+		t := target.NewTarget(gardenName, prod1Project.Name, "", prod1AmbiguousShoot.Name)
 		targetProvider := fake.NewFakeTargetProvider(t)
 
 		manager, err := target.NewManager(cfg, targetProvider, clientProvider, kubeconfigCache)
 		Expect(err).NotTo(HaveOccurred())
 		Expect(manager).NotTo(BeNil())
 
-		// Drop Project
-		Expect(manager.DropTargetProject()).Should(Equal(prod1Project.Name))
+		// Unset Project
+		Expect(manager.UnsetTargetProject()).Should(Equal(prod1Project.Name))
 
-		// should also drop seed and shoot
+		// should also unset seed and shoot
```

Comment:
in this test, seed was never targeted
```suggestion
		// should also unset shoot
```